1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,20 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+    let i = 0;
+    let str = "";
+    while (n > 0 && i < n)
+    {
+        if (i === 0)
+            str = "0";
+        else
+            str += ("," + 2*i);
+
+        i++;
+    }
+
+    console.log(str);
+
```

Comment:
I like the solution. Very clean. It seems you are quite advanced in coding considering we just started JS a few weeks back!